astoria/astprocd/usercode_lifecycle.py

```diff
@@ -207,6 +208,7 @@ def log(
         ) -> None:
             fh.write(data)
             fh.flush()
+            fsync(fh.fileno())
```

Comment:
Thank you for adding this, we had too many issues with logs not being synced to disk.